src/nvim/getchar.c

```diff
@@ -1063,9 +1065,10 @@ void del_typebuf(int len, int offset)
       typebuf.tb_off = MAXMAPLEN;
     }
     /* adjust typebuf.tb_buf (include the NUL at the end) */
+    size_t bytes;
+    STRICT_SUB(typebuf.tb_len, offset + 1, &bytes, size_t);
```

Comment:
Is it wrong here as well as #10508?

Reply:
Looks correct to me

`typebuf.tb_len - ( offset + 1) ==  (typebuf.tb_len +1) - offset`

Reply:
no, it's the same problem as https://github.com/neovim/neovim/commit/78f36fcf8c37bd51d81e9cf17de588a704a29769